src/py-opentimelineio/opentimelineio/schema/clip.py

```diff
@@ -23,11 +25,15 @@ def __repr__(self):
         'media_reference={}, '
         'source_range={}, '
         'metadata={}'
+        'effects={}'
+        'markers={}'
```

Comment:
Missing ", " separators like the pattern above.

Reply:
Apologies, bungled the changes, and committed too early.